src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/AuthnRequestAcceptanceTest.java

```diff
@@ -52,7 +52,7 @@
         ConfigOverride.config("server.connector.port", String.valueOf(0)),
         ConfigOverride.config("logging.loggers.uk\\.gov", "DEBUG"),
         ConfigOverride.config("samlSigningKey", TEST_RP_PRIVATE_SIGNING_KEY),
-        ConfigOverride.config("hubSsoLocation", String.format("%s/%s", COMPLIANCE_TOOL_HOST, "SAML2/SSO")),
+        ConfigOverride.config("verifyHubConfiguration.environment", "COMPLIANCE_TOOL"),
```

Comment:
Can we use the enum instead of the "COMPLIANCE_TOOL" string ?

Reply:
We have to provide a string to the config override (config is read from strings after all). Could set it to HubEnvironment.COMPLIANCE_TOOL.toString() (or .name()), but it seems more normal to put it in as it would appear in a config file.